cloud/performance.rst

```diff
@@ -60,10 +57,8 @@ Citus will output the progress as it moves each shard.
 
   The :code:`rebalance_table_shards` function rebalances all tables in the :ref:`colocation group <colocation_groups>` of the table named in its argument. Thus you do not have to call it for every single table, just call it on a representative table from each colocation group.
 
-.. _mx:
-
-Connections
-===========
+PgBouncer
```

Comment:
Should we rename this to Connection Scaling (pgBouncer) or some such? (Instead of just pgBouncer)